blues/simulation.py

```diff
@@ -224,6 +224,10 @@ def chooseMove(self):
 
     def simulateNCMC(self, verbose=False):
```

Comment:
>Am I understanding this correctly, in that the reporter will write to a DCD file before and after the move?

Pretty much, yes. It'll save the frame after the integration step and when a move is performed.